tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 requests
+scipy
```

Comment:
Note that `scipy` was added as a test dependency so we can compare the `com_sparse()` results with `scipy.ndimage.center_of_mass()` results.

Strangely, it seems that adding the `scipy` dependency added about 15 minutes to the Ubuntu CI time, because pip for python3.8 and python3.9 is having trouble finding/building a compatible scipy dependency for the tests. It eventually downgrades scipy a few times and finds one that works, but a lot of time is being spent trying to build newer versions of scipy, which fails. I don't know why it can't just install some of the `manylinux` scipy wheels that are already on pypi.

But, the tests still pass, even though it takes extra time.

Reply:
stempy also has stempy.image.com_dense(). That would be a lot easier with no need for new dependencies.

Reply:
Thanks Peter! I didn't realize we had that one. I just switched it out.